css/src/layout/system-admin--layout.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file
+ * Overrides 'Reusable layout' component of system/admin library.
+ */
+
+.layout-container {
+  margin-right: 1em;
+  margin-left: 1em;
+}
+
+@media screen and (min-width: 38em) {
```

Comment:
One more problem I found. This width seems quite narrow when combined with toolbar tray open in vertical mode
<img width="681" alt="screen shot 2019-01-03 at 21 20 15" src="https://user-images.githubusercontent.com/1845495/50642377-8ceaf000-0f9d-11e9-8dd3-b33d0bc34750.png">

Reply:
Good catch! I've say below 60em (960px), if the toolbar is vertical (`.toolbar-vertical`) we should use only one column.

Reply:
@ckrina Thanks for the prompt response, I'm on it.

Reply:
We don't have a vertical tray in Backdrop, but having that page switch to one-column on less wide screens is what we have also gone with, and it's been working quite well for us:

<img width="351" alt="screen shot 2019-01-04 at 8 32 34 am" src="https://user-images.githubusercontent.com/2423362/50676203-c9e5cf80-0ffb-11e9-893c-140bb36b048b.png">

So definitely 👍